upgrade/UPGRADE-v13.0.0-dev.md

```diff
@@ -46,3 +46,13 @@ There you can find links to upgrade notes for other versions too.
         - docker/php-fpm/docker-php-entrypoint
         - docker/php-fpm/phing-completion
     - see #project-base-diff to update your project
+- update your Elasticsearch Commands to support optional domainId
+    - `\Shopsys\FrameworkBundle\Command\Elasticsearch\AbstractElasticsearchIndexCommand`
+        - `executeForIndex()` changed its interface
+        ```diff
+            protected function executeForIndex(
+                OutputInterface $output,
+                AbstractIndex $index,
+        +       ?int $domainId = null
+            ): void
+        ```
```

Comment:
user story also mentions the need for updated documentation

Reply:
Well, there is no documentation about these commands, so I feel the commands docs in the command line are enough.

Reply:
It can be mentioned in https://docs.shopsys.com/en/13.0/introduction/console-commands-for-application-management-phing-targets/#elasticsearch-export as Tip "If you want to export only the one domain..."